science/projects/Ghazaleh Mahmoudi/01-main-content/main_content_detection.ipynb

```diff
@@ -0,0 +1,1400 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %%time
</pre><p>What is the purpose of calculating training time? </p><p>In my understanding, it would be better to calculate the test time, which I forget myself!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>To approximate the time required for training.</p><p>I also measure the time required to process the data for training.</p><p>I think we need to know these things in order to develop software and use it in the real usage.</p>